test/src/066-killall/main

```diff
@@ -3,9 +3,134 @@
 cvmfs_test_name="Killing all cvmfs2 instances"
 cvmfs_test_suites="quick"
 
+SRC="$(mktemp --suffix=FROZENFS)"
+DEST=/tmp/freezeme
+MOUNTPOINT=/cvmfs
+
+fs_setup(){
```

Comment:
Needs a comment explaining what is happening. I'd also  link to the blogpost.

Maybe we can start adding a "docstring" to the top that explains what the test is trying to do.